### PR TITLE
Do not show Wizard when BlueOS is down

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: core/frontend/src/components/wizard/Wizard.vue [hunk 1]
```diff
@@ -210,8 +210,8 @@ export default Vue.extend({
     },
   },
   async mounted() {
-    const wizard = await bag.getData('wizard')
-    if (wizard?.version !== WIZARD_VERSION) {
+    const wizard = await bag.getData('wizard2')
+    if (wizard !== undefined && wizard?.version !== WIZARD_VERSION) {
```
Comment (Williangalvani) [inline core/frontend/src/components/wizard/Wizard.vue:214]: what is the logic here? is undefined for "unable to reach backend"? I think it deserves at least a comment

File: core/frontend/src/components/wizard/Wizard.vue [hunk 2]
```diff
@@ -210,8 +210,8 @@ export default Vue.extend({
     },
   },
   async mounted() {
-    const wizard = await bag.getData('wizard')
-    if (wizard?.version !== WIZARD_VERSION) {
+    const wizard = await bag.getData('wizard2')
```
Comment (Williangalvani) [inline core/frontend/src/components/wizard/Wizard.vue:213]: why changing to "wizard2"?

**Review by Williangalvani — CHANGES_REQUESTED** [review]

File: core/frontend/src/components/wizard/Wizard.vue (hunk 2 above)
Comment (patrickelectric) [reply core/frontend/src/components/wizard/Wizard.vue:213]: Oh, that was code to test the feature

**Comment by Williangalvani** [comment]
fixes #1620 ?

**Pull request merged**